media/doc/3rd Party Files.txt

```diff
@@ -248,7 +248,7 @@ URL: http://www.mega-nerd.com/SRC/download.html
 Title: win-iconv
 Path: sdk/lib/3rdparty/libwin-iconv
 Used Version: git commit 8765259
```

Comment:
Note:
Since then, there were new commits, 2 releases, even newer commits.
A dev may want to upgrade this module.

Reply:
Filed as [CORE-19522](https://jira.reactos.org/browse/CORE-19522).